public/locales/en/translations.json

```diff
@@ -2,6 +2,15 @@
   "app": {
     "title": "Investor Portal: "
   },
+  "invest-popup": {
```

Comment:
Did you look at program-deposit component? We must distinguish the program deposit(investment) and the wallet deposit. In any case why you why did not you delete the old component?